megamek/src/megamek/client/commands/RulerCommand.java

```diff
@@ -31,7 +31,15 @@ public RulerCommand(Client client) {
         super(
                 client,
                 "ruler",
-                "Show Line of Sight (LOS) information between two points of the map. Usage: #ruler x1 y1 x2 y2 [elev1 [elev2]]. Where x1, y1 and x2, y2 are the coordiantes of the tiles, and the optional elev numbers are the elevations of the targets over the terrain. If elev is not given 1 is assumed which is for standing mechs. Prone mechs and most other units are at elevation 0.");
+                "Show Line of Sight (LOS) information between two points of the map. " +
+                        "Usage: #ruler x1 y1 x2 y2 [elev1 [elev2]]. " +
+                        "Where x1, y1 and x2, y2 are the coordinates of the tiles, and the optional elev " +
+                        "numbers are the elevations of the targets over the terrain. " +
+                        "If elev is not given 1 is assumed which is for standing mechs. " +
+                        "Prone mechs and most other units are at elevation 0. " +
+                        "cur hex can be given as the start hex, which will use the client's current hex. " +
+                        "eg; #ruler cur hex x2 y2, or as a short form #ruler x2 y2, " +
+                        "in which case elevation can't be used.");
```

Comment:
```suggestion
                "Show Line of Sight (LOS) information between two points of the map.\n" +
                        "Usage: #ruler x1 y1 x2 y2 [elev1 [elev2]].\n" +
                        "Where x1, y1 and x2, y2 are the coordinates of the tiles, and the optional elev " +
                        "numbers are the elevations of the targets over the terrain.\n" +
                        "If elev is not given 1 is assumed which is for standing mechs.\n" +
                        "Prone mechs and most other units are at elevation 0.\n" +
                        "cur hex can be given as the start hex, which will use the client's current hex.\n" +
                        "eg; #ruler cur hex x2 y2, or as a short form #ruler x2 y2,\n" +
                        "in which case elevation can't be used.");
```

Reply:
Hmm I don't think this is correct, the chat dondow already handles line wrapping, I just added new lines to break up the excessivily long string, to make it easier to edit.

These help strigns probalby should be moved to internationalization anyway right.

Reply:
I was just going from what MoveCommand had, but in any case just looking for consistency.

And yes, eventually these should be moved to localization.

Reply:
Move command had internal formatting because it had a complicated list of options, these other helps might be too long but they are single paragraphs not lists. The location of the newlines is purely for code readability and shouldn't show up in the help. For the move commands the new lines are there to make the command text readable. That seems quite different